src/web/modules/gateway_accounts/surcharge.njk

```diff
@@ -0,0 +1,54 @@
+{% from "components/input/macro.njk" import govukInput %}
+{% from "components/button/macro.njk" import govukButton %}
+
+{% extends "layout/layout.njk" %}
+
+{% block main %}
+  <span class="govuk-caption-m">{{service.name}} ({{ account.gateway_account_id }})</span>
+  <h1 class="govuk-heading-m">Corporate surcharge</h1>
+
+  <div>
+    <a href="/gateway_accounts/{{ account.gateway_account_id }}" class="govuk-back-link">Gateway account ({{ account.gateway_account_id }})</a>
+  </div>
+
+  <form method="POST" action="/gateway_accounts/{{ account.gateway_account_id }}/surcharge">
+    {{ govukInput({
+      id: "corporate_credit_card_surcharge_amount",
+      name: "corporate_credit_card_surcharge_amount",
+      value: account.corporate_credit_card_surcharge_amount or "0",
+      label: { text: "Corporate credit card surcharge amount" }
```

Comment:
Could you make this “Corporate non-prepaid credit card surcharge amount” for clarity?